### ISSUE TITLE
Project still active?

### ISSUE THREAD

**Issue by purposelycryptic** [ScudLee/anime-lists#277]
All activity seems to have suddenly stopped 3 weeks ago, including any mapping additions from open issues (or simply merging pull requests including them), and with GitHub showing zero activity by ScudLee since last November, it just seems like there might be an external issue not referenced anywhere?

I am unsure if this is the proper place to ask about this, but, seeing as other projects have this one as a dependency, and cannot work properly without it being updated regularly (HAMA, in my case), it is a real issue.

I am happy to contribute new mapping data as much as my own circumstances allow, but it would certainly help to know whether the project is still active, has migrated to another repo, or anything really regarding its current status.

Thank you for your patience with my selfish request.

**Comment by ifonefox** [comment]
ScudLee may be the owner of the repo, but isn't the only person with admin access to it. There are collaborators like Sixtenbe and slohj that can accept pull requests.

**Comment by purposelycryptic** [comment]
I was just wondering, since there has been no activity for 24 days now...

**Comment by ifonefox** [comment]
Sixtenbe merged and commented on many merge requests a couple days ago, so the project is definitely still active.

**Comment by purposelycryptic** [comment]
@ifonefox
Thanks for letting me know - figures all the pull requests except for mine and one other perfectly reasonable one would get merged.

Nothing like spending many, many hours looking up several dozen missing series from the last six months and beyond despite being too sick to even get out of bed, only to see your efforts go entirely ignored.

Sorry, my lungs feel like they're filled with broken glass and it's making me cranky, but you've been nothing but helpful, and I appreciate it.

I also think I'm done here - I only made a fork of the repo to contribute, but after over a month of no communication or any proper attempts by any of the maintainers to keep the lists up to date, I feel like I'm better off just switching everything dependent on this repo over to it instead. More work, but at least that way, if it appears abandoned, I have only myself to blame, and I can freely knock the person responsible upside the head. This project may not be dead, but it can't really be considered active either - certainly not to the extent of being relied on as a dependency.

I'm closing the issue, since I got my answer, and killing the pull request - seems like it was unwanted effort.

Anyway, thanks again for taking the time to reply to the issue, despite not being one of the folks actually maintaining the project.
